opm/material/fluidsystems/blackoilpvt/Co2GasPvt.hpp

```diff
@@ -111,12 +120,18 @@ class Co2GasPvt
      * \brief Returns the specific enthalpy [J/kg] of gas given a set of parameters.
      */
     template <class Evaluation>
-    Evaluation internalEnergy(unsigned,
+    Evaluation internalEnergy(unsigned regionIdx,
                         const Evaluation& temperature,
                         const Evaluation& pressure,
-                        const Evaluation&) const
+                        const Evaluation& rv,
+                        const Evaluation& rvw) const
     {
-        return CO2::gasInternalEnergy(temperature, pressure, extrapolate);
+        // assume ideal mixture
+        Evaluation result = 0;
+        const Evaluation xBrine = convertRvwToXgW_(Opm::max(rvw,rv),regionIdx);
```

Comment:
We're already in namespace Opm here, so you should either drop the namespace qualification (`max(rvw,rv)`) or use a globally anchored namespace qualification (`::Opm::max(rvw,rv)`).